rust/main/submitter/src/chain_tx_adapter/chains/ethereum/adapter.rs

```diff
@@ -170,4 +182,8 @@ impl AdaptsChain for EthereumTxAdapter {
     fn max_batch_size(&self) -> u32 {
         todo!()
     }
+
+    async fn tx_in_finality(&self, count: usize) {
```

Comment:
let's rename to `set_unfinalized_tx_count`, this initial name confused me

Reply:
Renamed